tests/js/web/PulsarUIComponentTest.js

```diff
@@ -17,14 +20,35 @@ describe('Pulsar UI Component', function() {
             <table class="table qa-table"></table>\
             <table class="table--datagrid qa-datagrid"></table>\
             <table class="table datatable qa-datatable"></table>\
+            <table class="table datatable table--horizontal qa-datatable-actions">\
+                <thead>\
+                    <tr>\
+                        <th>foo</th>\
+                        <th>bar</th>\
+                    </tr>\
+                </thead>\
+                <tbody>\
+                    <tr>\
+                        <td></td>\
+                        <td>\
+                            <div class="btn__group dropdown">\
+                                <button data-toggle="dropdown" class="btn dropdown__toggle row-actions">\
+                                    <i class="icon-ellipsis-h"><span class="hide">Actions</span></i>\
+                                </button>\
+                                <ul class="dropdown__menu pull-left">\
+                                    <li><a href="#action">action</a></li>\
+                                </ul>\
+                            </div>\
+                        </td>\
+                    </tr>\
+                </tbody>\
+            </table>\
```

Comment:
We could use some purdy ES6 template literals here and get rid of your escaped newlines `\`, it'd just be a case of replacing the single quotes for back ticks:

```
$('<div>\
    <p>Hello, World!</p>\
</div>')

$(`
    <div>
        <p>Hello, world!</p>
    </div>
`)
```